internal/cmd/auth.go

```diff
@@ -3,42 +3,41 @@ package cmd
 import (
 	"net/url"
 
+	"golang.org/x/xerrors"
+
 	"cdr.dev/coder-cli/coder-sdk"
 	"cdr.dev/coder-cli/internal/config"
-	"golang.org/x/xerrors"
 
 	"go.coder.com/flog"
 )
 
-// requireAuth exits the process with a nonzero exit code if the user is not authenticated to make requests
+// requireAuth exits the process with a nonzero exit code if the user is not authenticated to make requests.
 func requireAuth() *coder.Client {
 	client, err := newClient()
 	if err != nil {
-		flog.Fatal("%v", err)
+		flog.Fatal("%s", err)
```

Comment:
I've been meaning to remove this in favor of returning a proper error.

Reply:
https://github.com/cdr/coder-cli/issues/114